CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v1.1.9] 2020-05-04
```

Comment:
We kind of agreed to have release preparation in separate PR. @kopiczko suggests rationale for that is to make reverting changes easier. For that purpose probably even adding entry to Unreleased changelog update should be included in a separate PR from the one making the actual change, as reverting change commit should probably not revert changelog entry.

Reply:
Let me split that up. Would 1 PR for the change and one PR for the release notes do?

Reply:
I do not really follow this here. Too many information for me and I am not into the topic. In case I need to do anything here please be more specific.

Reply:
Guideline I was referring to is only for operators. It was shared in #news-dev via https://gigantic.slack.com/archives/C04TGHDEF/p1583408026036700
Work on enforcing it is covered in https://github.com/giantswarm/giantswarm/issues/9251

Reply:
I am still not sure what the question is or what I can do to help here.

Reply:
Thanks for the guidelines Stevo :)

Tim, I think Stevo just mentionned you to explain why and who is enforcing this separate PR guidelines and to give more context if needed.
He did a pretty good job to explain the rationale and challenges so I do not thing you need to help :)

Reply:
> For that purpose probably even adding entry to Unreleased changelog update should be included in a separate PR from the one making the actual change, as reverting change commit should probably not revert changelog entry.

I think only release header should be added separately. Entries should be added with the feature/fix. When feature/fix is reverted then the entries will get removed. As long as something is not released this is what we want.